### PR TITLE
prepare for v1.19.1

### PR DESCRIPTION
**Pull request by jackjia-ibm**
Signed-off-by: Jack (T.) Jia <jack-tiefeng.jia@ibm.com>

### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/getting-started/summaryofchanges.md [hunk 1]
```diff
@@ -30,6 +31,20 @@ Zowe Version 1.19 and earlier releases include the following enhancements, relea
 - [Version 1.0.0 (February 2019)](#version-1-0-0-february-2019)
 
 
+## Version 1.19.1 LTS (February 2021)
+
+### Notable changes 
+
+**SMPE PTF**
+
+With the continuos optimization of Zowe build size, now Zowe release can fit in one PTF. This single PTF UO01969 will supersede PTF UO01967 and UO01968 from v1.19.0 release.
```
Comment (OnnoVdT) [inline docs/getting-started/summaryofchanges.md:40]: typo: continuos -> continuous

File: docs/getting-started/summaryofchanges.md [hunk 2]
```diff
@@ -30,6 +31,20 @@ Zowe Version 1.19 and earlier releases include the following enhancements, relea
...
+
+### Bug Fixes
+
+**SMPE PTF**
+
+Fixed missing HOLD data about extra steps the system programmer should take after applying the PTF. It explains new configurations we introduced in `instance.env` and also a reconfiguration step with `zowe-configure-instance.sh` is mandatory.
```
Comment (OnnoVdT) [inline docs/getting-started/summaryofchanges.md:46]: new configurations -> new configuration options

File: docs/README.md [hunk 3]
```diff
@@ -39,9 +39,9 @@ footer: Except where otherwise noted, content on this site is licensed under a C
 
 ### Zowe documentation
 
-You can download the Version 1.x.x Zowe documentation in PDF format from the links below. The latest version on this website is 1.19.0.
+You can download the Version 1.x.x Zowe documentation in PDF format from the links below. The latest version on this website is 1.19.1.
 
-**[V1.19.0](https://docs.zowe.org/stable/Zowe_Documentation.pdf)** | 
+**[V1.19.x](https://docs.zowe.org/stable/Zowe_Documentation.pdf)** |
```
Comment (OnnoVdT) [inline docs/README.md:44]: is it indeed v1.19.x ? other places you always specify 1.19.1

**Review by OnnoVdT — CHANGES_REQUESTED** [review]

**Review by OnnoVdT — APPROVED** [review]

**Comment by nannanli** [comment]
@jackjia-ibm Thank you for the updates! I just made very minor tweaks. It looks good to me now so feel free to merge it when 1.19.1 is published.

**Review by nannanli — APPROVED** [review]

**Pull request merged**